api.py

```
@@ -0,0 +1,733 @@
+    if meetingbaas_bot_id:
+        # Start the Pipecat process for this bot
+        pipecat_websocket_url = f"ws://localhost:8766/pipecat/{bot_client_id}"
+
+        # Update the converter's sample rate to match the streaming audio frequency
+        sample_rate = 24000 if request.streaming_audio_frequency == "24khz" else 16000
+        converter.set_sample_rate(sample_rate)
+        logger.info(
+            f"Set audio sample rate to {sample_rate} Hz for {request.streaming_audio_frequency}"
+        )
+
+        process = start_pipecat_process(
```

Ruff (0.8.2)

Local variable `process` is assigned to but never used

Remove assignment to unused variable `process`

(F841)